staging/src/k8s.io/apiserver/pkg/authentication/token/cache/cache_simple.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cache
+
+import "sync"
+
+// simple key/value cache with read/write locking
+type simpleCache struct {
```

Comment:
> I don't actually want an LRU cache... just the TTL. An LRU cache turns every read into a write.

That means I'm going to find later on in this pull a spot where you're pruning old entries?  You really think that's better?  Seems like it would be pretty trivial to explode memory size.